site2/docs/concepts-messaging.md

```diff
@@ -916,7 +937,9 @@ The other available approach to message deduplication is to ensure that each mes
 
 Message deduplication makes Pulsar an ideal messaging system to be used in conjunction with stream processing engines (SPEs) and other systems seeking to provide effectively-once processing semantics. Messaging systems that do not offer automatic message deduplication require the SPE or other system to guarantee deduplication, which means that strict message ordering comes at the cost of burdening the application with the responsibility of deduplication. With Pulsar, strict ordering guarantees come at no application-level cost.
 
-> You can find more in-depth information in [this post](https://www.splunk.com/en_us/blog/it/exactly-once-is-not-exactly-the-same.html).
+:::tip
+You can find more in-depth information in [this post](https://www.splunk.com/en_us/blog/it/exactly-once-is-not-exactly-the-same.html).
```

Comment:
```suggestion

```
Suggest removing this. 
To keep neutral and not to be challenged by ASF, we avoid adding blogs of commercial companies to the Pulsar documentation.

Reply:
OK. Removed.